arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -3579,6 +3772,13 @@
           "$ref": "#/definitions/Object",
           "readOnly": true,
           "description": "Gets the tracked properties."
+        },
+        "retryHistory": {
```

Comment:
Is it intentional that we're adding properties without bumping the version of the spec?